nrf-sdc/src/sdc.rs

```diff
@@ -1482,6 +1490,7 @@ pub mod vendor {
     sdc_cmd!(NordicBigReservedTimeSet => sdc_hci_cmd_vs_big_reserved_time_set(x));
     sdc_cmd!(NordicCigReservedTimeSet => sdc_hci_cmd_vs_cig_reserved_time_set(x));
     sdc_cmd!(NordicCisSubeventLengthSet => sdc_hci_cmd_vs_cis_subevent_length_set(x));
+    sdc_cmd!(NordicSetEventStartTask => sdc_hci_cmd_vs_set_event_start_task(x));
```

Comment:
I think we're missing `sdc_cmd!` invocations for the other vendor specific commands?